### DISCUSSION TITLE
push access denied after "personal access token (classic)" regenerated

### DISCUSSION THREAD

**Discussion by Papa-26** [community/community#69949] [Repositories] [answered]
### Select Topic Area

General

### Body

Roughly every 2 months I am asked to regenerate my PAT(classic), which I do. Since the most recent one, I am unable to push recent project changes on one of my projects. I have tried every one of of my past PATs, and all end in 'access denied'.
What should I do?

Username for 'https://github.com': xxxxx
Password for 'https://xxxxxx@github.com': 
remote: Permission to Papa-26/yyyyy.git denied to xxxxxx.
fatal: unable to access 'https://github.com/xxxx/yyyyy/': The requested URL returned error: 403

**Comment by thevindu-w** [comment]
May be you can try creating a new token and use that. Make sure to add the necessary permissions to the PAT (in your case, write access to repo)

**Reply by Papa-26** [reply]
I am a bit confused about the whole PAT business. I have two kind of projects: Organization projects (where I happen to be responsible for the organization), and private projects, that have nothing to do with the organization. Up to now I have used the same PAT for both projects. My problem is with a private project. How do I untangle the access privileges?

**Reply by thevindu-w** [reply, marked as answer]
You can view/change the access privileges for tokens at [https://github.com/settings/tokens](https://github.com/settings/tokens). It should have repo privileges. For example, it can look like,
![image](https://github.com/community/community/assets/91218789/1e218d21-7387-4fde-845a-f456978753ac)

**Reply by Papa-26** [reply]
Thank you thevindu-w for your help. It worked perfectly, and I am able to function again. It was very generous of you, to offer your time to fix my problem.
Papaworx-26
